Visma.net/Models/Inventory.cs

```diff
@@ -34,7 +34,12 @@ public string baseUnit
             set => Set(value);
         }
 
-        [JsonProperty] public CostPriceStatistics costPriceStatistics { get; private set; }
```

Comment:
This needs to be changed back in order for it to be correct. CostPriceStatistics cannot be set in the api, just read.